cms/djangoapps/contentstore/views/course.py

```diff
@@ -1159,7 +1161,12 @@ def settings_handler(request, course_key_string):
                         # None is chosen, so remove the course prerequisites
                         course_milestones = milestones_api.get_course_milestones(course_key=course_key, relationship="requires")
                         for milestone in course_milestones:
-                            remove_prerequisite_course(course_key, milestone)
+                            ee_milestone_namespace = generate_milestone_namespace(
+                                get_namespace_choices().get('ENTRANCE_EXAM'),
+                                course_key
+                            )
+                            if not milestone["namespace"] == ee_milestone_namespace:
```

Comment:
Please use the more explicit operator `x != y` than the `not x == y`:

```suggestion
                            if milestone["namespace"] != generate_milestone_namespace(
                                    get_namespace_choices().get('ENTRANCE_EXAM'), course_key
                            ):
```